### PR TITLE
Add scripts for Jellyfish prow jobs

### PR DESCRIPTION
**Pull request by nesmabadr**
**Description:**
Changes proposed in this pull request:

- Add scripts to be used by the unstable-cli prow job

**Related issue(s):**
Part of https://github.com/kyma-project/lifecycle-manager/issues/682

### CODE REVIEW

File: scripts/prow/cli-integration.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+apk add nodejs npm
+install_dir="/usr/local/bin"
+mkdir -p "$install_dir"
+pushd "$install_dir" || exit
+curl -Lo kyma.tar.gz "https://github.com/kyma-project/cli/releases/latest/download/kyma_linux_x86_64.tar.gz" && tar -zxvf kyma.tar.gz && chmod +x kyma && rm -f kyma.tar.gz
+kyma version --client
+popd || exit
+kyma provision k3d --ci
+kyma deploy --ci --concurrency=8 --profile=evaluation --source=main
+make -C "../../kyma-project/kyma/tests/fast-integration" "ci"
```
Comment (Ressetkk) [inline scripts/prow/cli-integration.sh:11]:
```suggestion
make -C "../kyma/tests/fast-integration" "ci"
```

File: scripts/prow/cli-integration-release-upgrade.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+kyma version --client
+popd || exit
+kyma provision k3d --ci
+kyma deploy --ci --concurrency=8 --profile=evaluation --source="${KYMA_SOURCE}" --verbose
+git remote add origin https://github.com/kyma-project/kyma.git
+git reset --hard && git remote update && git fetch --all >/dev/null 2>&1 && git checkout "${KYMA_SOURCE}"
+make -C "../../kyma-project/kyma/tests/fast-integration" "ci-pre-upgrade"
+kyma deploy --ci --concurrency=8 --profile=evaluation --source="${KYMA_UPGRADE_VERSION}" --verbose
+git reset --hard && git remote update && git fetch --all >/dev/null 2>&1 && git checkout "${KYMA_UPGRADE_VERSION}"
+make -C "../../kyma-project/kyma/tests/fast-integration" "ci-post-upgrade"
```
Comment (Ressetkk) [inline scripts/prow/cli-integration-release-upgrade.sh:20]:
```suggestion
make -C "../kyma/tests/fast-integration" "ci-post-upgrade"
```

File: scripts/prow/cli-integration-release-upgrade.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+set -e
+apk add nodejs npm
+kyma_get_previous_release_version_return_version=$(curl --silent --fail --show-error -H "Authorization: token ${BOT_GITHUB_TOKEN}" "https://api.github.com/repos/kyma-project/kyma/releases" | jq -r 'del( .[] | select( (.prerelease == true) or (.draft == true) )) | sort_by(.tag_name | split(".") | map(tonumber)) | .[-2].target_commitish | split("/") | .[-1]')
```
Comment (Ressetkk) [inline scripts/prow/cli-integration-release-upgrade.sh:3]: Do you need to call GitHub API to get latest version? Kyma is tagged semantically and actually you have the complete information about Git state once it's cloned.

It'd be beneficial if you could drop usage of GitHub token and rely on native git calls.

File: scripts/prow/cli-integration-release-upgrade.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+mkdir -p "$install_dir"
+pushd "$install_dir" || exit
+curl -Lo kyma.tar.gz "https://github.com/kyma-project/cli/releases/latest/download/kyma_linux_x86_64.tar.gz" && tar -zxvf kyma.tar.gz && chmod +x kyma && rm -f kyma.tar.gz
+kyma version --client
+popd || exit
+kyma provision k3d --ci
+kyma deploy --ci --concurrency=8 --profile=evaluation --source="${KYMA_SOURCE}" --verbose
+git remote add origin https://github.com/kyma-project/kyma.git
+git reset --hard && git remote update && git fetch --all >/dev/null 2>&1 && git checkout "${KYMA_SOURCE}"
+make -C "../../kyma-project/kyma/tests/fast-integration" "ci-pre-upgrade"
```
Comment (Ressetkk) [inline scripts/prow/cli-integration-release-upgrade.sh:17]:
```suggestion
make -C "../kyma/tests/fast-integration" "ci-pre-upgrade"
```

File: scripts/prow/cli-integration.sh (hunk 1 above)
Comment (nesmabadr) [reply scripts/prow/cli-integration.sh:11]: the reconciler is in the Kyma-incubator not Kyma-project that's why we need this

Comment (Ressetkk) [reply scripts/prow/cli-integration.sh:11]: Oh yeah, correct. Then ignore that suggestion

**Review by Ressetkk — DISMISSED** [review]
Nthing to add so far.

**Review by lindnerby — CHANGES_REQUESTED** [review]

**Review by lindnerby — APPROVED** [review]

**Pull request merged**